timeseries-streaming/timeseries-python-applications/MLPipeline/timeseries/encoder_decoder/encoder_decoder_run_fn.py

```diff
@@ -82,39 +88,38 @@ def run_fn(fn_args: TrainerFnArgs):
     print(f"Parameters {fn_args}")
 
     train_dataset = _input_fn(
-            fn_args.train_files,
-            tf_transform_output,
-            batch_size=fn_args.train_batches)
+        fn_args.train_files,
+        tf_transform_output,
+        batch_size=fn_args.train_batches)
 
     eval_dataset = _input_fn(
-            fn_args.eval_files,
-            tf_transform_output,
-            batch_size=fn_args.eval_batches)
+        fn_args.eval_files,
+        tf_transform_output,
+        batch_size=fn_args.eval_batches)
 
     # mirrored_strategy = tf.distribute.MirroredStrategy()
     # with mirrored_strategy.scope():
-    model = encoder_decoder_model._build_keras_model(
-            tf_transform_output=tf_transform_output,
-            timesteps=fn_args.timesteps,
-            number_features=fn_args.number_features,
-            outer_units=fn_args.outer_units,
-            inner_units=fn_args.inner_units)
+    model = encoder_decoder_model.build_keras_model(
+        timesteps=fn_args.timesteps,
+        number_features=fn_args.number_features,
+        outer_units=fn_args.outer_units,
+        inner_units=fn_args.inner_units)
 
     model.fit(
-            train_dataset,
-            epochs=fn_args.epochs,
-            steps_per_epoch=fn_args.train_steps,
-            validation_data=eval_dataset,
-            validation_steps=fn_args.eval_steps)
+        train_dataset,
+        epochs=fn_args.epochs,
+        steps_per_epoch=fn_args.train_steps,
+        validation_data=eval_dataset,
+        validation_steps=fn_args.eval_steps)
 
     signatures = {
-            'serving_default': _get_serve_tf_examples_fn(
-                    model, tf_transform_output).get_concrete_function(
-                            tf.TensorSpec(
-                                    shape=(None),
-                                    dtype=tf.string,
-                                    name='examples')),
+        'serving_default': _get_serve_tf_examples_fn(
+            model, tf_transform_output).get_concrete_function(
+            tf.TensorSpec(
+                shape=None,
```

Comment:
in other [examples](https://github.com/tensorflow/tfx/blob/master/tfx/examples/iris/iris_utils_native_keras.py#L256) it's all set to [None], Is None correct?

Reply:
Changed, although looks like both None and [None] are ok.